src/Slider/Slider.stories.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Slider from "./Slider";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  argTypes: {
+    value: { type: "number" }
+  },
+  component: Slider,
+  title: "General/Slider"
+};
+
+const Template = args => {
+  const [value, setValue] = React.useState(args.value);
+  React.useEffect(() => {
+    setValue(args.value);
+  }, [args.value]);
+  const onChange = (event, value) => {
+    setValue(value);
+    action("onChange")(event, value);
+  };
+  return (
+    <div style={{ width: 500 }}>
+      <Slider {...args} onChange={onChange} value={value} />
+    </div>
+  );
+};
+
+// default story
+export const Default = Template.bind({});
+Default.args = {
+  displayCurrentValue: "auto",
+  labels: true,
+  max: 10,
+  min: 1,
+  step: 1,
+  title: "Slide example"
+};
+
+// custom labels story
+export const CustomLabels = Template.bind({});
+CustomLabels.args = {
+  displayCurrentValue: "auto",
+  labels: true,
```

Comment:
This is confusing, especially with predefValues. I would expect labels to be the cusotm labels I pass in whereas it's actually a boolean to show/hiode labels. Thus I would rename labels to showLabels or displayLabels to make it clear it's a boolean. I would then rename predefValues to labels.